### ISSUE TITLE
[help] configure log task with load-grunt-config

### ISSUE THREAD

**Issue by erredeco** [xavierpriour/grunt-log#1]
Hi, I have installed your plugin and I use load-grunt-config, but I am not able to configure it properly
I created  a log.js file under /grunt/config

the file now is pretty empty:

```
module.exports = {

}
```

If I run grunt log:msg it writes "msg", but now I want to store the log messages into variables and output them. How can I do? Thank you for your support

**Comment by xavierpriour** [comment]
If your message is in variable `myMsg`, call `"log:"+myMsg` instead of `"log:msg"`

**Comment by erredeco** [comment]
@xavierpriour thank you for your answer;
ummm I have some doubts: I usually use the aliases.yaml file to define the tasks like:

```
build:
 - 'copy:all'
- 'uglify'
- log:whatever
```

I don't know how to write something like "log:"+myMsg there
regards

**Comment by xavierpriour** [comment]
I don't think you can use variables in yaml file - you should switch to a Javascript aliases file - `aliases.js`, as shown [on load-grunt-config](https://github.com/firstandthird/load-grunt-config#aliases)
